### PR TITLE
Update zh.json

### PR DESCRIPTION
**Pull request by Cmax-aka-67349**

### CODE REVIEW

File: lang/zh.json [hunk 1]
```diff
@@ -776,81 +776,81 @@
       "condition": "避开滚球道中的所有保龄球并抵达保龄瓶世界"
     },
     "jade": {
-      "name": "Jade Sky",
-      "description": "Jade beings and jade structures",
-      "condition": "Visit the Jade Sky Hamlet"
+      "name": "碧玉之空",
+      "description": "玉石的生物，玉石的建筑",
```
Comment (nekocarpet) [inline lang/zh.json:780]: 翡翠城下，玉石苍生

File: lang/zh.json [hunk 2]
```diff
@@ -776,81 +776,81 @@
...
+      "condition": "抵达翡翠镇"
     },
     "sea_star": {
-      "name": "Sea Star",
-      "description": "Fun in the sun?",
-      "condition": "Fill the Calcarina Sea"
+      "name": "海星",
+      "description": "阳光下的嬉戏？",
```
Comment (nekocarpet) [inline lang/zh.json:785]: `    "description": "我觉得海星？",`

File: lang/zh.json [hunk 3]
```diff
@@ -776,81 +776,81 @@
...
     "heart_3": {
-      "name": "Heart of Life",
-      "condition": "Reach 999 Heart Points from vending machines (granted upon waking up)"
+      "name": "生命之心",
+      "condition": "通过在自动贩卖机处购买食品将生命值提升至999，然后捏脸醒来"
     },
     "square_ruins": {
-      "name": "Memories of the Old Ruins",
-      "condition": "Find the old ruins leading to the Viridian Wetlands"
+      "name": "远古遗迹的回忆",
```
Comment (nekocarpet) [inline lang/zh.json:813]: `     "name": "古迹追忆",`

File: lang/zh.json [hunk 4]
```diff
@@ -776,81 +776,81 @@
...
+      "condition": "找到通往常绿湿地的古老遗迹"
     },
     "egg": {
-      "name": "Memories of the Chromatic Nest",
-      "condition": "Obtain menu themes 8-15 and 25. For any you have already obtained, return to the area where you originally found them."
+      "name": "七彩之巢的回忆",
```
Comment (nekocarpet) [inline lang/zh.json:817]: `   "name": "彩巢回忆",`

File: lang/zh.json [hunk 5]
```diff
@@ -776,81 +776,81 @@
...
     "mini_town": {
-      "name": "Giant",
-      "condition": "Enter Mini-Town"
+      "name": "巨人",
+      "condition": "进入迷你城"
     },
     "gakuran": {
-      "name": "Wandering Boy",
-      "condition": "Interact with Gakuran-kun while using a certain effect"
+      "name": "徘徊的男孩",
```
Comment (nekocarpet) [inline lang/zh.json:833]: `     "name": "彷徨少年",`

File: lang/zh.json [hunk 6]
```diff
@@ -776,81 +776,81 @@
...
+      "condition": "用某个效果与学兰君互动"
     },
     "block_person": {
-      "name": "Block Person",
-      "description": "I've turned into a block person?!",
-      "condition": "Transform into a block person in the Red Brick Maze"
+      "name": "方块人",
+      "description": "我居然变成了一个方块人？！",
```
Comment (nekocarpet) [inline lang/zh.json:838]: `      "description": "我方了？！",`

File: lang/zh.json [hunk 7]
```diff
@@ -776,81 +776,81 @@
...
+      "condition": "在红砖迷宫里变身方块人"
```
Comment (nekocarpet) [inline lang/zh.json:839]: `      "condition": "在红砖地牢里变身方块人"`

File: lang/zh.json [hunk 8]
```diff
@@ -776,81 +776,81 @@
...
     },
     "madotsukis_room": {
-      "name": "Original Likeness",
```
Comment (nekocarpet) [inline lang/zh.json:842]: `      "name": "既视感",`

File: lang/zh.json [hunk 9]
```diff
@@ -776,81 +776,81 @@
...
-      "description": "It clearly resembles a certain place",
-      "condition": "Enter a certain room in the Never-Ending Hallway"
+      "name": "原汁原味",
```
Comment (nekocarpet) [inline lang/zh.json:842]: `      "name": "既视感",`

File: lang/zh.json [hunk 10]
```diff
@@ -776,81 +776,81 @@
...
+      "description": "明显与某个地方很像",
```
Comment (nekocarpet) [inline lang/zh.json:843]: `      "description": "喷涌而出的既视感",`

File: lang/zh.json [hunk 11]
```diff
@@ -776,81 +776,81 @@
...
+      "condition": "进入无尽走廊里的某个房间"
     },
     "haniwa": {
-      "name": "Haniwa Dance",
-      "description": "Look at her go!",
-      "condition": "View the Haniwa Dance event at the Haniwa Temple"
+      "name": "埴轮之舞",
+      "description": "快看她的舞蹈！",
```
Comment (nekocarpet) [inline lang/zh.json:848]: `      "description": "快看她跳！",`

File: lang/zh.json [hunk 12]
```diff
@@ -892,9 +892,9 @@
       "condition": "在与公寓墙壁的瞪眼比赛中获胜"
     },
     "other_worlds": {
-      "name": "Other Worlds",
-      "description": "The piano chords lead... somewhere...",
-      "condition": "Enter the Young Passage and the Microscopic World"
+      "name": "其他的世界",
```
Comment (nekocarpet) [inline lang/zh.json:895]: `      "name": "他世",`

File: lang/zh.json [hunk 13]
```diff
@@ -949,9 +949,9 @@
       "condition": "进入埃舍尔花园"
     },
     "moonrise": {
-      "name": "Moonrise",
-      "description": "What a view!",
-      "condition": "Reach the Jovian Moon"
+      "name": "明月初升",
+      "description": "风景真不错！",
```
Comment (nekocarpet) [inline lang/zh.json:953]: `      "description": "把酒问青天",`

File: lang/zh.json [hunk 14]
```diff
@@ -1039,20 +1039,20 @@
...
     "eye_flower": {
-      "name": "Eye Flower",
-      "condition": "Use the Seer effect near a certain flower in Alien Plants World to make it bud"
+      "name": "眼之花",
+      "condition": "在外星植物世界中，于某朵花旁边使用先知效果使其发芽"
     },
     "arcade": {
-      "name": "Bright Arcade",
-      "condition": "View all the active arcade machines in the Concrete Treehouse"
+      "name": "明亮街机",
```
Comment (nekocarpet) [inline lang/zh.json:1046]: `      "name": "幻光街机",`

File: lang/zh.json [hunk 15]
```diff
@@ -776,81 +776,81 @@
...
-      "description": "Look at her go!",
-      "condition": "View the Haniwa Dance event at the Haniwa Temple"
+      "name": "埴轮之舞",
+      "description": "快看她的舞蹈！",
+      "condition": "在埴轮神庙里观看埴轮之舞"
     },
     "snowflake": {
-      "name": "Snowy Traveler",
-      "condition": "Visit the Snowy Village"
+      "name": "雪白旅人",
```
Comment (nekocarpet) [inline lang/zh.json:852]:
```suggestion
      "name": "风雪旅人",
```
或者白雪也可以

**Review by nekocarpet — COMMENTED** [review]
Well done! Just a few suggestions from me.

File: lang/zh.json (hunk 7 above)
Comment (Cmax-aka-67349) [reply lang/zh.json:839]: 昨天问了一下群友，发现“红砖迷宫”这个称呼更常见一些

**Comment by Cmax-aka-67349** [comment]
These are brilliant suggestions!

**Review by Flashfyre — APPROVED** [review]

**Pull request merged**